src/Elmish.WPF/ViewModel.fs

```diff
@@ -126,25 +126,16 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
         r
 
   let measure2 name callName f =
-    if not config.Measure then f
-    else
-      fun x y ->
-        let sw = System.Diagnostics.Stopwatch.StartNew ()
-        let r = f x y
-        sw.Stop ()
-        if sw.ElapsedMilliseconds >= int64 config.MeasureLimitMs then
-          log "[%s] %s (%ims): %s" propNameChain callName sw.ElapsedMilliseconds name
-        r
+    fun x -> measure name callName (f x)
```

Comment:
Are you 100% sure this is behaviour preserving? Multi-parameter functions and partial application can bite you in some cases (see e.g. https://github.com/elmish/Elmish.WPF/issues/33#issuecomment-530536977)

Reply:
Of course the correctness is the same, but I admit that I am not 100% sure that the performance is the same.  I will look into this later.  I am aware of the gotcha one can fall into memoizing a partially applied function.

In the meantime, how about this alternative instead?